sdk/monitor/monitor-ingestion/README.md

```diff
@@ -244,6 +244,8 @@ Logs uploaded using the Monitor Ingestion client library can be retrieved using
 
 ## Troubleshooting
 
+For details on diagnosing various failure scenarios, see our [troubleshooting guide](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-ingestion/TROUBLESHOOTING.md)
+
 ### Logging
```

Comment:
Remove this logging section, as it's already covered in the troubleshooting guide.

Reply:
done